.github/workflows/prerelease.yaml

```diff
@@ -52,3 +53,19 @@ jobs:
             ]'
           prerelease: true
           release-as: ${{ steps.extract_branch.outputs.branch }}
+      - name: Extract prerelease version
+        shell: bash
+        run: echo "version=$(echo "${{ steps.extract_branch.outputs.branch }}" | cut -d'/' -f2)" >> $GITHUB_OUTPUT
+        id: extract_version
+      - name: Trigger dispatch event
+        uses: peter-evans/repository-dispatch@v3
+        # release please run 2 times, first for creating the PR and second for creating the release
+        if: ${{ steps.release_action.outputs.releases_created == 'true' }}
```

Comment:
Didn't really understand. the event trigger that's configured is a commit on prerelease/* branch. why will this run for release?

Reply:
when we create the new branch this action will run and create a [pull req](https://github.com/rudderlabs/rudder-server/pull/4911) with all the change logs:
https://github.com/rudderlabs/rudder-server/actions/runs/10037166229
Once we merge that [PR](https://github.com/rudderlabs/rudder-server/pull/4911) this action will run again and create a release
https://github.com/rudderlabs/rudder-server/actions/runs/10037739692

Reply:
make sense. I missed the new PR 🤦 ... I was thinking of only new commits that are added to prerelease/* branch